test/solver.jl

```diff
@@ -309,10 +309,12 @@ k = 1
         @test norm(d_RR[n+1:end,:]) < tol*norm(A)
         @test size(d_F) == size(A)
         @test size(d_F.Q, 1) == size(A, 1)
+        @test det(d_F.Q) ≈ det(collect(d_F.Q * CuMatrix{elty}(I, size(d_F.Q)))) atol=tol*norm(A)
```

Comment:
@maleadt I think this version addressed the indexing issue https://github.com/JuliaGPU/CuArrays.jl/pull/391#issuecomment-522935578